### ISSUE TITLE
Cavalry Death Charge

### ISSUE THREAD

**Issue by jamnub2** [duke-tylerjone/gc-bugs-and-feedback#3]
Have like a Cavalry death charge key like z or something and it makes the horse go faster and the sabre will one hit on impact but after the charge your horse will slow down for 30 seconds and it will be on a cooldown of 5 minutes to avoid abuse
Cons: you can't turn
You can only swing your sabre once or twice in the charge and the cooldowns
Pros:
1 hit as stated
Horse moves faster 
And it does ram damage like Infantry dies or ragdolls
(repost)

**Comment by Corruptikoo** [comment]
Realistically I don't think you could control a horse to suicide ram into a group of infantry, unfortunately.

**Comment by jamnub2** [comment]
Yeah but horses did death charge Infantry in some battles and it would be a way to stop Infantry to just block and get away with 0 reprisal it would also add new strategies to the game and to make Infantry more careful when dealing with Cavalry
